package.json

```diff
@@ -5,7 +5,7 @@
   "main": "lib/index.js",
   "scripts": {
     "test": "npm run lint && npm run test:unit",
-    "test:unit": "mocha test/test_*.js",
+    "test:unit": "mocha test/test_*.js && rm -r test/fixtures/dist",
```

Comment:
This is a bad solution... But was a quick fix for now. I think this should happen in the afterEach Block?

Reply:
👨‍💻🛋 would argue that you want those artifacts to stick around for inspectability - but yeah, feel free to tear it down (perhaps in `after` rather than `afterEach` for efficiency: _at the moment_, there's only a single test that actually generates files IIRC).

Reply:
Interesting. But for me this lead to wrong green tests, as files were left behind from a previous run and I expected on those 🤔

Reply:
Yeah, fair; premature optimization - so `afterEach` it is.

Reply:
Done. Had to use an [NPM package](https://www.npmjs.com/package/rimraf) for that. I... don't understand why this is not in the standard library.

Reply:
https://twitter.com/rickhanlonii/status/1062319416107560961

Reply:
😂